Core/HLE/sceRtc.cpp

```diff
@@ -459,16 +459,14 @@ static int sceRtcConvertLocalTimeToUTC(u32 tickLocalPtr,u32 tickUTCPtr)
 	{
 		u64 srcTick = Memory::Read_U64(tickLocalPtr);
 		// TODO : Let the user select his timezone / daylight saving instead of taking system param ?
-#if defined(__GLIBC__) || defined(BLACKBERRY) || defined(__SYMBIAN32__)
+#ifndef _MSC_VER
```

Comment:
`tm_gmtoff` seems to have appeared somewhere between 4.3BSD and 4.4BSD-Lite. A few ancient Unices haven't inherited it (AIX, IRIX, HPUX). Solaris have and newer libcs come with `tm_gmtoff` as well (musl, bionic).